tests/tasks/sinopia/test_local_metadata.py

```diff
@@ -31,7 +33,7 @@ def mock_airflow_variables(monkeypatch):
     def mock_get(*args, **kwargs):
         if args[0].startswith("sinopia_user"):
             return "ils_middleware"
-        if args[0].startswith("sinopia_api_uri"):
+        if args[0].startswith("dev_sinopia_api_uri"):
```

Comment:
Since we'll be running Airflow instances for dev, stage, and prod environments, this should be `sinopia_api_uri`